src/main/java/pl/mateusz/zadanie20/UserRepository.java

```diff
@@ -2,13 +2,19 @@
 
 import org.springframework.stereotype.Repository;
 
-import java.util.ArrayList;
 import java.util.List;
 
 @Repository
 public class UserRepository {
 
-    private List<User> userList = new ArrayList<>();
+    private List<User> userList;
```

Comment:
Czy jak IntelliJ podpowiada żeby zrobić: private final List<User> userList; to zmieniać?

Reply:
i podobnie IntelliJ podpowiada w klasie HelloController mianowicie: private final UserRepository userRepository; czy zmieniać w takich przypadkach?

Reply:
Tak, generalnie przy wstrzykiwaniu zależności warto używać final, To takie zabezpieczenie, że przez przypadek nie nadpiszemy pola po wstrzyknięciu go przez Springa